packages/react/package.json

```diff
@@ -74,7 +74,7 @@
     "ts-loader": "^9.4.3",
     "typed-css-modules": "0.9.1",
     "typescript-plugin-css-modules": "5.1.0",
-    "webpack": "5.97.1",
+    "webpack": "5.101.3",
```

Comment:
Do we need to call out that we've bumped Webpack in the changelog, given that it's responsible for generating our final bundle?

Reply:
Thanks for reminder, this was definitely missing a changeset for the Node version 😅. We need to spin the minor when we do these. To your question, there was no diff to the output here luckily, but we should add it just in case it does on future PR's. Added to same changeset 👍